frontend/src/app/main/ui/workspace/sidebar/options/menus/component.cljs

```diff
@@ -643,5 +730,123 @@
 
           (when (and (not swap-opened?) (not multi))
             [:& component-annotation {:id id :shape shape :component component :rerender-fn rerender-fn}])
+
+          (when (and is-variant? (not swap-opened?) (not multi))
+            [:> component-variant* {:component component :shape shape :libraries libraries}])
+
           (when (dbg/enabled? :display-touched)
             [:div ":touched " (str (:touched shape))])])])))
+
+
+(mf/defc variant-menu
```

Comment:
New components should follow the new naming and call convention: name ends with `*` and instantiate with `:>` instead of `:&`  (also, no metada is required with that components)